### PR TITLE
US138101: [CERT] Monthly HTML Editor Bump - 20.22.5

### PR DESCRIPTION
**Pull request by prajapac**
Add insert-practice and content-styler attributes to HTML editor component

Rally Link: https://rally1.rallydev.com/#/289692574792d/custom/355050439968?detail=%2Fuserstory%2F631700478359

### CODE REVIEW

File: components/d2l-activity-editor/d2l-activity-quiz-editor/d2l-activity-quiz-add-activity-menu.js [hunk 1]
```diff
@@ -291,7 +291,7 @@ class ActivityQuizAddActivityMenu extends ActivityEditorMixin(SkeletonMixin(Loca
 		}
 		return html`
 			<d2l-menu-item
-				text="${ltiProviders[0].name()}"
+				text="${ltiProviders[0]?.name()}"
```
Comment (ianlaidlaw) [inline components/d2l-activity-editor/d2l-activity-quiz-editor/d2l-activity-quiz-add-activity-menu.js:294]: what's the purpose behind this change? does this repo support optional chaining?

Comment (prajapac) [reply components/d2l-activity-editor/d2l-activity-quiz-editor/d2l-activity-quiz-add-activity-menu.js:294]: Genuinely not sure where that change came from, I have to head out for the day but will investigate/fix in the morning!

Comment (kristian-torres) [reply components/d2l-activity-editor/d2l-activity-quiz-editor/d2l-activity-quiz-add-activity-menu.js:294]: I think activities does support optional chaining (and if it doesn't, can someone show me how to check?). This change came from here: https://github.com/BrightspaceHypermediaComponents/activities/commit/9e3c417a3224ab574c076ac30255315703cdd28f

seems unrelated but I'm curious why it shows up as a diff on your PR.

Comment (ianlaidlaw) [reply components/d2l-activity-editor/d2l-activity-quiz-editor/d2l-activity-quiz-add-activity-menu.js:294]: I wonder if this was in master when Chirag checked out but it has since been reverted and Chirag just needs to pull? Only thing I can think of

**Review by kristian-torres — APPROVED** [review]

File: components/d2l-activity-editor/d2l-activity-quiz-editor/d2l-activity-quiz-add-activity-menu.js (hunk 1 above)
Comment (prajapac) [reply components/d2l-activity-editor/d2l-activity-quiz-editor/d2l-activity-quiz-add-activity-menu.js]: Ahh yes, that could be it. Should I create another PR with that, or is this PR good as is? Also, re-ran CI because tests failed for some reason.

Comment (prajapac) [reply components/d2l-activity-editor/d2l-activity-quiz-editor/d2l-activity-quiz-add-activity-menu.js]: YOLO
